statusboard/src/components/ProjectsTable/TopicsFilter.js

```diff
@@ -3,6 +3,16 @@ import React from 'react';
 
 import Select from '../Select/Select';
 
+const emptyOptionText = '- Select -';
+
+export function filterByTopics(rows, id, filterValue) {
+  if (filterValue === emptyOptionText) {
+    return rows;
+  }
+  const valueKey = id[0];
+  return rows.filter((r) => r.values[valueKey].search(filterValue) !== -1);
```

Comment:
I think this should split by commas and do an `indexOf`, otherwise if you select the tag "code" you'll end up getting "code-for-whatever" as it also contains the substring.